cekit/descriptor/image.py

```diff
@@ -22,7 +22,8 @@
   modules: {type: any}
   packages: {type: any}
   osbs: {type: any}
-  volumes: {type: any}""")
+  volumes: {type: any}
+  help_template: {type: text}""")
```

Comment:
I'm not sure if we would like to have something like dictionary here so we can pass more parameteres in future..

Reply:
Ah perhaps a help: key and a dict underneath with template: and possibly other stuff in future?